src/modules/sensors/vehicle_imu/VehicleIMU.cpp

```diff
@@ -699,15 +699,26 @@ void VehicleIMU::SensorCalibrationUpdate()
 					const bool valid = (hrt_elapsed_time(&estimator_sensor_bias.timestamp) < 1_s) &&
 							   (estimator_sensor_bias.accel_device_id != 0) &&
 							   estimator_sensor_bias.accel_bias_stable &&
-							   (bias_variance.max() < max_var_allowed) &&
-							   (bias_variance.max() < max_var_ratio * bias_variance.min());
+							   (bias_variance.min() > min_var_allowed) &&
+							   (bias_variance.max() < max_var_allowed);
```

Comment:
This is now different. 
Before:
1. variance < some max value
2. ratio between all variances < 100 (i.e.: variance "condition" check)

After:
1. variance < some max value
2. variance > some max value / 100

Meaning that we won't save the bias if they all converge properly but below the min value.

Reply:
Thanks, I misread it.

Reverting.